index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" type='text/css'>
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <nav>
+                <img class="img img1" src="left_topside_icon.svg" alt="Image geometry on the left top side">
+                <img class="img img2" src="ic_menu.svg" alt="Menu image on the right top side">
+            </nav>
+        </header>
+        <section class="textcontainer">
+            <h1 class="text text1">Charlie Giorgania</h1>
+            <p class="text text2">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </section>
+        <section class="rectangle">
+            <h4>Start collaboration</h4>
```

Comment:
- [ ] `Start collaboration` should be a `<button>` or an `<a>` and given button interaction effects like hover e.t.c. Please work on this. See a screenshot of that requirement below 👇🏼
    
 ![buttons](https://user-images.githubusercontent.com/45002654/125559383-0b4a8b3c-c13c-4a67-adf9-d78033bcdcbd.PNG)

Reply:
I have removed the h4 tag and implemented the <a> functionality with hover functionality.